GloVe/download_glove_840b_300d.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+echo "Downloading GloVe Common Crawl (840B tokens, 2.2M vocab, cased, 300d vectors, 2.03 GB download)"
+wget http://nlp.stanford.edu/data/glove.840B.300d.zip
```

Comment:
@tuzhucheng why don't we script it up in Python? We can then use something like, `hashlib.md5()`